package.json

```diff
@@ -8,7 +8,8 @@
     "release": "standard-version"
   },
   "dependencies": {
-    "raw-loader": "~0.5.1"
+    "raw-loader": "~0.5.1",
+    "loader-utils": "^1.2.3"
```

Comment:
Nee update `package-json.lock` too and we can merge :+1:

Reply:
Installing it with `npm i -P` seems to change `package-lock.json` quite a bit (mainly changing packages versions from `x.y.z` to `^x.v.w`. Should I commit it anyway or just add the part related to `loader-utils` ?

Reply:
@aymasse yep, need commit with package-json.lock

Reply:
Good to be merged?